### ISSUE TITLE
False positives when linting workflows that use reusable workflows

### ISSUE THREAD

**Issue by sammcj** [cschleiden/actions-linter#123]
Reusable workflows do not have:

- `runs-on`
- `steps`

and have a top level `uses`.

For example - here is a completely valid workflow that calls a reusable workflow:

```
jobs:
  lint-actions-workflows:
    uses: mycompany/reusable-workflows-if/.github/workflows/lint-actions-workflows.yml@main
```

However the actions-linter fails:

![Screen Shot 2022-05-04 at 2 31 51 pm](https://user-images.githubusercontent.com/862951/166623471-6576ad9a-61d9-484f-b9a5-ce93929b16b5.jpg)
